### ISSUE TITLE
The bot counts itself as a voting member in the vote to skip a song

### ISSUE THREAD

**Issue by dihur** [DasWolke/rem-v2#11]
Steps to reproduce:

1. Create a new server.
2. Join the voice channel of that server.
3. Add Rem Bot to that server.
4. !w.play [a song]
5. !w.skip

Output:

Type !w.yes to voteskip the current song. Current votepercentage at 50%/50
Users that have voted:

.------------------------.
| the_only_member |
'-------------------------'

But clearly, if the only human member of the voice channel has voted, then 100% of the members, whose vote should matter, have voted.

Image:
![bug](https://cloud.githubusercontent.com/assets/24232445/21458960/eae6f33a-c93a-11e6-9ffe-108f77ffc2e0.png)

**Comment by DasWolke** [comment]
That is clearly an issue, let me hotfix that real fast

**Comment by dihur** [comment]
I have examined your commit, however I am not sure it completely fixes the bug.

If my understanding is correct, the bug lies in this line of the function startVoteskip:
60	let size = channel.voiceMembers.size;

The bot is counted, because it is in the voice channel, as it is playing the song. Having already assured that there are more than zero voice channel members before calling startVoteskip, we should also decrement the variable size so as not to count the music-playing bot:
60	let size = channel.voiceMembers.size - 1;

The more general fix would be not counting all the bots, but I doubt that many bots are in the voice channel at the same time in practice (since the only reason a bot would join the voice channel that I know of is to play music).

To show that the bug also happens when there is more than one voice channel member (in the image I have attached in the original post, the bot was not in the voice channel, because the song had already finished):

![bug2](https://cloud.githubusercontent.com/assets/24232445/21460016/22b45bbe-c944-11e6-948e-f90160592e08.png)


The percentage should be 50%, because of the two members of the voice chat that are not Rem Bot, one has voted to skip the song.
